Source/dotnet/Samples/ValidationApp/TestService.cs

```diff
@@ -87,8 +88,8 @@ public async Task<TestResult> TestDisplay()
 
         _display = new DisplayService(
             new DisplayScreen(
-                _hardware.Display,
-                Meadow.Peripherals.Displays.RotationType._270Degrees,
+                (IPixelDisplay)_hardware.Display,
```

Comment:
I don't think you need the cast here now

Reply:
I do, because DisplayScreen is expecting a `IPixelDisplay` and not a `IColorInvertableDisplay`: https://github.com/WildernessLabs/Meadow.Foundation/blob/develop/Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics.MicroLayout/Driver/DisplayScreen.cs